### ISSUE TITLE
Allow RES&MAG on same character (Initiation/Submersion tab)

### ISSUE THREAD

**Issue by Vereaux** [chummer5a/chummer5a#3407]
**Describe the bug**
Bit of an odd ball bug, considering what it is that I'm trying to do with the program. Under normal circumstances, I imagine this bug would never occur in character creation (I think, but I could be wrong). In this, I'm trying to create a Mystic Adept who is also a Technomancer. Technically against established canon rules, but this is a home-brewed character, so we're ignoring that detail for various reasons of our own. 

For the most part, the program works just as intended with what we are doing. It lists the spells, adept powers, complex forms, etc. However, if I try to do anything with Initiation or Submersion, I get one tab, but not the other. As this is a character who would need both tabs, I'm stuck. 

**To Reproduce**
1. Have enabled "Allow Initiation/Submersion in Create Mode" under House Rules in the Options menu.
2. Create a new character with default settings, but check "Ignore Character Creation Rules" and choose "Mystic Adept" {This is merely what I initially started with, but I've found starting with either a magical type or Technomancer does not change the outcome}. 
3. Add Technomancer Quality. 

**Expected behavior**
I expected both tabs to be available, but the Initiation Tab is replaced by Submersion instead. 

**Desktop (please complete the following information):**
 - OS: Windows 7
 - Version: 5.204.40

**Additional context**
Again, not a normal case of character creation, but I figured I might as well ask. Worst case scenario, I could be told no.

**Comment by chummer5a** [comment]
There aren't two tabs, there's one tab with different text labels based on if you have magic or resonance. There's some discussion of moving the forms into individual user controls that would more easily allow this sort of thing, but it's currently not possible.

**Comment by Vereaux** [comment]
Yeah, I just realized that right before I saw your reply. I was trying different configurations with the program, seeing if they would show up in similar locations (kinda cheat it in a way, make them both appear in a print out at least if the program wouldn't let me see it in the interface), but when I realized it was the same tab and used the same text fields, I was like "He'd have to rewrite all the code for that tab...damn...", so yeah @.@

What I'll do in the mean time is calculate the Technomancer information in a separate character file, and just keep track of both. Not the best solution, but considering the amount of work you'd have to do otherwise, I believe I can wait =P

Thank you for getting back to me so quickly. I honestly thought this wouldn't have gotten a look until tomorrow, as it's after midnight here =P
